xchange-btcmarkets/src/main/java/org/knowm/xchange/btcmarkets/BTCMarketsAuthenticated.java

```diff
@@ -43,16 +43,6 @@ BTCMarketsOrders getOpenOrders(
       BTCMarketsOpenOrdersRequest request)
       throws BTCMarketsException, IOException;
 
-  @POST
-  @Path("order/history")
-  @Consumes(MediaType.APPLICATION_JSON)
-  BTCMarketsOrders getOrderHistory(
-      @HeaderParam("apikey") String publicKey,
-      @HeaderParam("timestamp") SynchronizedValueFactory<Long> nonceFactory,
-      @HeaderParam("signature") BTCMarketsDigest signer,
-      BTCMarketsOpenOrdersRequest request)
-      throws BTCMarketsException, IOException;
```

Comment:
if its DTOs are not incorrect I think its worth leaving and exposing it at least via the *Raw service

Reply:
Why would we leave in an old implementation that's no longer in use?, that API will eventually be deprecated. clearing out the old client code gives a good overview of how far along the migration is.

Reply:
if its getting deprecated then ok, lets remove it